addons/mail/static/src/core/web/activity_button.js

```diff
@@ -88,6 +88,7 @@ export class ActivityButton extends Component {
                 onActivityChanged: (thread) => {
                     const recordToLoad = resIds ? selectedRecords : [this.props.record];
                     recordToLoad.forEach((r) => r.load());
+                    this.postOnActivityChanged();
```

Comment:
I find the "post" in the name confusing, particularly in this scope where "post" is "sending" something.

Couldn't it just be `onActivityChanged`? With potentially the full code moved into the method?

Reply:
I think he meant to extract the code inside the prop function to the newly created method and give it as prop to the popover. Unless I misunderstood something.